.github/workflows/release.yaml

```diff
@@ -0,0 +1,59 @@
+name: Release
+
+on:
+  workflow_call:
+    inputs:
+      config-name:
+        required: true
+        type: string
+    secrets:
+      PAT:
+        required: true
+  
+permissions:
+  contents: read
+
+jobs:
+  initial_release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+    - name: Git checkout
+      uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Enable python
+      uses: actions/setup-python@v3
+
+    - name: Run python script
+      env:
+          GITHUB_TOKEN: ${{ secrets.PAT }} 
+          GITHUB_REPOSITORY: ${{env.GITHUB_REPOSITORY}}
+      run: |
+           pip install requests
+           python3 ./.github/workflows/scripts/initial_release.py
```

Comment:
Same I mentioned above, how would this file be distributed and updated on the executing repository?